app/backend/src/couchers/servicers/threads.py

```diff
@@ -29,6 +29,7 @@ def unpack_thread_id(thread_id: int) -> (int, int):
 class Threads(threads_pb2_grpc.ThreadsServicer):
     def GetThread(self, request, context):
         database_id, depth = unpack_thread_id(request.thread_id)
+        request.page_size = request.page_size if request.page_size < 100000 else None  # upper bound
```

Comment:
This line has no effect, remove. There's no point in changing the request structure, as it is only inputs to the function. Use local variables instead.